clouddriver-ecs/src/main/java/com/netflix/spinnaker/clouddriver/ecs/provider/EcsProvider.java

```diff
@@ -30,11 +34,9 @@
   private static final Map<String, String> urlMappingTemplates = new HashMap<>();
```

Comment:
Could you call this class an `EcsCacheProvider`?  Or it would break conventions?

Reply:
Yeah, it's a conventional thing.

See:
AwsProvider
DockerRegistryProvider
KubernetesProvider
AppengineProvider